examples/js/loaders/GLTFLoader.js

```diff
@@ -1246,6 +1246,11 @@ THREE.GLTFLoader = ( function () {
 
 		// Invalid URL
 		if ( typeof url !== 'string' || url === '' ) return '';
+		
+		// Host Relative URL
+		if (/^https?:\/\//i.test(path) && /^\//.test(url)) {
```

Comment:
Could you give an example of a URL (or a model) with the issue you’re trying to solve? Something like `/texture.png`? This seems to be testing for URLs that begin with both `http` and `/` which would be mutually exclusive.

Reply:
Oh, I missed that this was testing the path and then the URL separately. Still, an example would be helpful.

Reply:
Here is an example of a model - https://res.cloudinary.com/du46dspul/image/upload/f_gltf,fl_draco/gpm2qrjnowle2cvge8we

Reply:
Ok thanks, this looks good to me. Here and below could you replace match code style with spaces inside parens and around if blocks?

```js
if ( /^https?:\/\//i.test( path ) && /^\//.test( url ) ) {

  path = path.replace( /(^https?:\/\/[^\/]+).*/i, '$1' );

}
```

Reply:
done